saleor/product/filters.py

```diff
@@ -79,3 +77,23 @@ def _get_product_variants_attributes_filters(self):
 
     def _get_attribute_choices(self, attribute):
         return [(choice.pk, choice.name) for choice in attribute.values.all()]
+
+    def validate_sort_by(self, value):
+        if value.strip('-') not in SORT_BY_FIELDS:
+            raise ValidationError(
+                ('%s is not an even number' % value)
```

Comment:
Is this the right message? Also, error messages should be marked for translation as they are displayed in the UI.

Reply:
Wow, correct in progress.

Reply:
Done